packages/design-system-react-native/package.json

```diff
@@ -72,11 +69,14 @@
     "@types/react": "^18.2.0",
     "@types/react-test-renderer": "^18",
     "babel-jest": "^29.7.0",
+    "bitcoin-address-validation": ">=2.0.0",
```

Comment:
We specify `bitcoin-address-validation` with the version constraint `>=2.0.0` to ensure compatibility with MetaMask Mobile, which currently uses version `2.0.0` ([reference](https://github.com/MetaMask/metamask-mobile/blob/main/package.json#L295)). This range also permits upgrades to future major versions, including `3.x.x`, since both v2 and v3 are compatible with design system libraries. Notably, v3.0.0 introduces ESM (ECMAScript Modules) support, which this project fully supports, as reflected in the [CHANGELOG](https://github.com/ruigomeseu/bitcoin-address-validation/blob/dbfd2db43cca1c29d3978180958e06566711baa5/CHANGELOG.md).